clang/test/CodeGenSPIRV/Builtins/ids_and_ranges.c

```diff
@@ -2,12 +2,18 @@
 // RUN: %clang_cc1 -O1 -triple spirv64 -cl-std=CL3.0 -x cl %s -emit-llvm -o - | FileCheck %s --check-prefixes=CHECK,CHECK64
 // RUN: %clang_cc1 -O1 -triple spirv32 -cl-std=CL3.0 -x cl %s -emit-llvm -o - | FileCheck %s --check-prefixes=CHECK,CHECK32
 
+#ifdef __SYCL_DEVICE_ONLY__
+#define SYCL_EXTERNAL [[clang::sycl_external]] __attribute__((sycl_device))
```

Comment:
`__attribute__((sycl_device))` is not needed here due to the changes to `getLLVMLinkageForDeclarator()`, right?

Reply:
Correct. Completely unnecessary - just missed it here. Thanks for the catch!